README.md

```diff
@@ -7,6 +7,26 @@
 
 The Music Encoding Initiative (MEI) is an open-source effort to define a system for encoding musical documents in a machine-readable structure. MEI brings together specialists from various music research communities, including technologists, librarians, historians, and theorists in a common effort to define best practices for representing a broad range of musical documents and structures. The results of these discussions are formalized in the MEI schema, a core set of rules for recording physical and intellectual characteristics of music notation documents expressed as an eXtensible Markup Language (XML) schema. It is complemented by the MEI Guidelines, which provide detailed explanations of the components of the MEI model and best practices suggestions.
 
+# Structure of this repository
+
+This repository contains all the source code of the MEI Schema and Guidelines:
+
+ * .github: Configuration files for Github Actions workflows.
+ * customizations: TEI ODD files that allow you to build customized MEI schemata.
+ * source: Contains the source code, as expressed in TEI ODD. This includes the source code for the MEI Guidelines, and the MEI schema modules.
+ * submodules: A container for Git Submodules, that is third party developments that are needed for e.g. building this repository's contents, but not part of our codebase.
+ * tests: Unit tests for the MEI Schemata.
+ * utils: Helper scripts e.g. for compiling schemata and guidelines from this repository.
+
+**!ATTENTION!**
+
+The prebuild schema files no longer exist in this repository, since reintegrating the guidelines and configuring a continuous integration workflow the build artifacts of this repository (Schemata and Guidelines) are being generated on every commit to the develop branch and automatically pushed to separate repositories:
+
+* Schema: https://github.com/music-encoding/schema
+* Guidelines: https://github.com/music-encoding/guidelines
```

Comment:
There are no links in the Markdown, but GH will display them as links, right?

Reply:
yup